src/server/redis_connection.h

```diff
@@ -71,6 +71,17 @@ class Connection : public EvbufCallbackBase<Connection> {
   std::string Double(double d) const {
     return protocol_version_ == RESP::v3 ? "," + util::Float2String(d) + CRLF : BulkString(util::Float2String(d));
   }
+  // ext is the extension of file to send, 'txt' for text file, 'md ' for markdown file
+  // at most 3 chars, padded with space
+  // if RESP is V2, treat verbatim string as blob string
+  // https://github.com/redis/redis/blob/7.2/src/networking.c#L1099
+  std::string VerbatimString(std::string ext, const std::string &data) const {
+    CHECK(ext.size() <= 3);
+    int padded_len = 3 - static_cast<int>(ext.size());
+    ext = ext + std::string(padded_len, ' ');
+    return protocol_version_ == RESP::v3 ? "=" + std::to_string(3 + data.size() + 1) + CRLF + ext + ":" + data + CRLF
```

Comment:
```suggestion
    return protocol_version_ == RESP::v3 ? "=" + std::to_string(3 + 1 + data.size()) + CRLF + ext + ":" + data + CRLF
```

Reply:
`3+1+data.size()` describes the format `ext:data` better. thanks